app/src/main/java/ch/epfl/sdp/drone3d/model/mission/MappingMission.kt

```diff
@@ -21,14 +22,15 @@ enum class State {
 }
 
 /**
- * A MappingMission is instantiated with a [name], a [flightPath] and a [flightHeight].
+ * A MappingMission is instantiated with a [name], a [flightHeight], a [strategy] and a [area].
  * The [privateId], [sharedId] and [state] are updated according to where the MappingMission are stored.
  * The [ownerUid] is set the first time the mission is either stored or shared.
  */
 data class MappingMission(
     val name: String = "",
-    val flightPath: List<LatLng> = listOf(),
-    val flightHeight:Double = 0.0,
+    val flightHeight: Double = 0.0,
+    val strategy: Strategy = Strategy.SINGLE_PASS,
+    val area: List<LatLng> = listOf(),
```

Comment:
I don't see why we set some default values,
Like for the name and the area, why would we want to not specify them ?

Reply:
Because it needs to have an empty constructor for firebase to be able to serialize it.